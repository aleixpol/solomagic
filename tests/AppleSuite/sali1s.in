\ref something0
\mt somethingsomething salinaha somethingsomething

\ref something1
\mt somethingsomething nanaha somethingsomething

