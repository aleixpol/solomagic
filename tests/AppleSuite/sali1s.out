\ref something0
\mt somethingsomething salinaha somethingsomething
\ma somethingsomething salingnaha somethingsomething

\ref something1
\mt somethingsomething nanaha somethingsomething
\ma somethingsomething nanaha somethingsomething
